test/suite/test_durable_rollback_to_stable.py

```diff
@@ -44,7 +44,8 @@ class test_durable_rollback_to_stable(wttest.WiredTigerTestCase, suite_subproces
     keyfmt = [
         ('row-string', dict(keyfmt='S')),
         ('row-int', dict(keyfmt='i')),
-        ('column-store', dict(keyfmt='r')),
+    # The commented columnar tests needs to be enabled once rollback to stable for columnar is fixed.
```

Comment:
Can you create a ticket to enable these or if there is already one refer that ticket in the comment here.

Reply:
Done. Added Jira ticket number WT-5548